Assets/02. Scripts/02-20. SceneLogic/WaitingScene/PlayersInfo.cs

```diff
@@ -0,0 +1,69 @@
+using Mirror;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class PlayersInfo : MonoBehaviour
+{
+    [SerializeField]
+    private List<UI_PlayerInfoSlot> PlayerInfoSlotList;
```

Comment:
- 
```suggestion
    private List<UI_PlayerInfoSlot> _playerInfoSlotList;
```
왜 제미나이가 이걸 못잡아줬는지 모르겠네용..